docs/build-on-polkadot/xcm/hrmp-channels/para-to-para.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Opening HRMP Channels Between Parachains
+description: Learn how to open HRMP channels between parachains on Polkadot. Discover the step-by-step process for establishing uni- and bidirectional communication.
+---
+
+# Opening HRMP Channels Between Parachains
+
+## Introduction
+
+For establishing communication channels between parachains on the Polkadot network using the HRMP protocol, the following steps are required:
```

Comment:
```suggestion
For establishing communication channels between parachains on the Polkadot network using the Horizontal Relay-routed Message Passing (HRMP) protocol, the following steps are required:
```
Let's ensure we spell out acronyms like this the first time they appear on a page. If the user didn't start with the Overview, they still wouldn't know what this means when they start reading this page.